common/components/filter/template.njk

```diff
@@ -0,0 +1,17 @@
+<nav class="app-filter">
```

Comment:
Looks like we're missing some tests for this component. Take a look at some other examples for how we test different scenarios.